samples/graph-change-notification-team-channel-nodejs/nodejs/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "graph-notification",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
```

Comment:
Why do we have 2 package.json and 2 package-lock.json ??

Remove one which is not in use.